### PR TITLE
fix(error-page): add status code

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
SSIA

### CODE REVIEW

File: src/common/pages/ErrorPage.vue [hunk 1]
```diff
@@ -5,10 +5,15 @@
...
+                {{ statusCode }}
             </h2>
             <h3 class="error-message">
-                {{ $t('COMMON.ERROR.404_MSG') }}
+                <template v-if="statusCode === '403'">
+                    {{ $t('COMMON.ERROR.404_MSG') }}
+                </template>
+                <template v-else>
+                    {{ $t('COMMON.ERROR.404_MSG') }}
+                </template>
```
Comment (piggggggggy) [inline src/common/pages/ErrorPage.vue:16]: question: Both cases seem to be same?

**Review by piggggggggy — APPROVED** [review]
LGTM!~!
I left small comment

File: src/common/pages/ErrorPage.vue (hunk 1 above)
Comment (WANZARGEN) [reply src/common/pages/ErrorPage.vue:16]: Yes. This is just to distinguish the case for internal users and developers.
There is no planning. I just added it 😅

Comment (piggggggggy) [reply src/common/pages/ErrorPage.vue:16]: Oh I see... 🫡

**Pull request merged**